scripts/dup_audit.py

```
@@ -0,0 +1,530 @@
+    for pat in EXCLUDE_PATTERNS:
+        if pat in path:
+            return True
+    return False
+
+
+def list_target_files() -> List[str]:
+    files: List[str] = []
+    for d in TARGET_DIRS:
+        if not os.path.isdir(d):
+            continue
+        for root, dirs, filenames in os.walk(d):
```

Ruff (0.12.2)

Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)